### PR TITLE
DNM 21297 - Edit.js: Make the default particle system attractive

### PR DESCRIPTION
**Pull request by Triplelexx**
PR for feedback.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0d21d7108e35f7f26708d8cb2109005aec88bfd9 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-0d21d71.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-0d21d71.dmg

File: scripts/system/edit.js [hunk 1]
```diff
@@ -482,22 +482,52 @@ var toolBar = (function () {
...
+                radiusFinish: 0.25,
+                radiusSpread: 0.5,
+                alpha: 0,
+                alphaStart: 0.5,
                 alphaFinish: 0,
-                emitRate: 100,
-                textures: "https://hifi-public.s3.amazonaws.com/alan/Particles/Particle-Sprite-Smoke-1.png"
+                polarStart: 0,
+                polarFinish: 0,
+                textures: "https://raw.githubusercontent.com/Triplelexx/hifi/40908fb1a31bd884f87e21c4e524275f8334471b/interface/resources/images/Smoke.png"
```
Comment (thoys) [inline scripts/system/edit.js]: I've uploaded your smoke image on the S3,  you can replace the link with this: 
https://hifi-content.s3.amazonaws.com/DomainContent/production/Particles/Smoke.png

File: scripts/system/edit.js [hunk 2]
```diff
@@ -2062,7 +2092,7 @@ function selectParticleEntity(entityID) {
 
     selectedParticleEntity = entityID;
     particleExplorerTool.setActiveParticleEntity(entityID);
-    particleExplorerTool.webView.emitScriptEvent(JSON.stringify(particleData));    
+    particleExplorerTool.webView.emitScriptEvent(JSON.stringify(particleData));
```
Comment (thoys) [inline scripts/system/edit.js]: You made the code look more attractive as well 👍

Comment (Triplelexx) [inline scripts/system/edit.js]: 🤖

**Comment by hifi-gustavo** [comment]
Build console output for commit: fae51c5c4652b2bb623168243968b0474f91c4a7 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-fae51c5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-fae51c5.dmg

**Comment by PhilipRosedale** [comment]
Emission rate and velocity seems too high.  Is this some other bug?  I'd just like to see it look a bit slower... like the smoke coming from a fire.  It seems to be 'shooting' up into the air so fast there is almost a strobing effect.

**Comment by Triplelexx** [comment]
@PhilipRosedale I was going for something like smoke from a train, I'll try some changes later today then.

**Comment by Triplelexx** [comment]
Sorry, didn't have time today to focus on this again. 😞 I'll get on it asap.

**Comment by hifi-gustavo** [comment]
Build console output for commit: f98c3556fd0011150e35dc20275434b6321ac014 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-f98c355.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-f98c355.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 90432562df729c8961ab297bacc698562309ba0b available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-9043256.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-9043256.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7edda8733027c4df9340edd76570141659e13298 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-7edda87.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-7edda87.dmg

**Comment by PhilipRosedale** [comment]
I really want to get this right - have something that when a new user creates one they will say "cool!".  So... this texture (or it's size or something) is almost completely invisible to me.  I have to turn carefully to be able to see it at all.  Is there any way to make it more visible?  Can more particles be emitted?  Also, there appears to be a repeating 'pulse' to it... is there any way to make it more random?

**Comment by Triplelexx** [comment]
@PhilipRosedale I think it may be easier to find a suitable effect if someone could advise in world. Would that be possible?

**Comment by ZappoMan** [comment]
test this please

**Comment by YuppyPlays** [comment]
Can test, Test plan?

**Comment by ZappoMan** [comment]
@YuppyPlays - "test this please" is a command to tell the build system to build the PR again... no need for QA at this point. Sorry.

**Comment by YuppyPlays** [comment]
Oh right!, was not aware of this, thanks for letting me know and no worries :)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a742002e26738023f1b41a4cea75a460eb53665 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-5a74200.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-5a74200.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5a742002e26738023f1b41a4cea75a460eb53665 available at: https://gist.github.com/5ed95027add3dd432688b9b0533f7c17<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-5a74200.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10381-5a74200.dmg

**Comment by PhilipRosedale** [comment]
Texture needs to be checked in, yes?   Otherwise looks good.

**Comment by Triplelexx** [comment]
Thanks. Yes, will update with production location if someone can upload the new one.

**Comment by thoys** [comment]
Here @Triplelexx , https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png

**Pull request merged**